### ISSUE TITLE
valid-expect does not handle "not" assertions properly

### ISSUE THREAD

**Issue by alecxe** [tlvince/eslint-plugin-jasmine#104]
This looks like a bug to me. The [`valid-expect`](https://github.com/tlvince/eslint-plugin-jasmine/blob/master/docs/rules/valid-expect.md) rule is complaining about a matcher not being called if there is a `.not` type assertion:

```js
expect("text1").not.toEqual("text2");
```

Using the latest `eslint-plugin-jasmine`. Can work on a fix. Thanks.

**Comment by burabure** [comment]
yeah, this is an error, can you submit a PR?

**Comment by alecxe** [comment]
@burabure yeah, sorry, that was my mistake, I will add tests and a fix. Thanks.
